specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-11-01/consumption.json

```diff
@@ -0,0 +1,5397 @@
+          },
+          "ChargesListByBillingProfile-Modern": {
+            "$ref": "./examples/ChargesListByModernBillingProfile.json"
+          },
+          "ChargesListByBillingProfileInvoiceSection-Modern": {
+            "$ref": "./examples/ChargesListByModernBillingProfileInvoiceSection.json"
+          },
+          "ChargesListByBillingProfileGroupByInvoiceSectionId-Modern": {
+            "$ref": "./examples/ChargesListByModernBillingProfileGroupByInvoiceSectionId.json"
+          },
+          "ChargesListByInvoiceSectionId-Modern": {
+            "$ref": "./examples/ChargesListByModernInvoiceSectionId.json"
+          },
+          "ChargesListByCustomer-Modern": {
+            "$ref": "./examples/ChargesListByModernCustomer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeChargesParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "startDate",
+            "in": "query",
+            "description": "Start date",
+            "required": false,
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          },
+          {
+            "name": "endDate",
+            "in": "query",
+            "description": "End date",
+            "required": false,
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          },
+          {
+            "name": "$filter",
+            "description": "May be used to filter charges by properties/usageEnd (Utc time), properties/usageStart (Utc time). The filter supports 'eq', 'lt', 'gt', 'le', 'ge', and 'and'. It does not currently support 'ne', 'or', or 'not'. Tag filter is a key value pair string where key and value is separated by a colon (:).",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$apply",
+            "description": "May be used to group charges for billingAccount scope by properties/billingProfileId, properties/invoiceSectionId, properties/customerId (specific for Partner Led), or for billingProfile scope by properties/invoiceSectionId.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ChargesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.Consumption/balances": {
```

Comment:
So, why call out this path specifically?  Is this the only allowed path for the 'balances' extension resource, or can it be at any scope?

Reply:
Yes it is the only allowed path